doc/modules/cassandra/pages/managing/operating/onboarding-to-accord.adoc

```diff
@@ -0,0 +1,347 @@
+= Onboarding to Accord
+
+== Intro
+
+Accord supports all existing CQL and can be enabled on a per table and
+per token range within that table basis. Enabling Accord does require a
+migration process that can be done on this same per table and per range
```

Comment:
to clarify, should this be `Enabling Accord on existing tables` as I would expected `CREATE TABLE blah WITH transactional_mode = 'full'` to not require any migration (and `IF NOT EXISTS` should if the table exists leaving the transactional mode alone).